client/composables/tokens/useCreateToken.ts

```diff
@@ -0,0 +1,13 @@
+import type { MessageDTO } from '~/utils/messages'
```

Comment:
maybe we should put all those "service-y" composables in a dedicated "composable/services" folder

Reply:
I don't get it... Can you be more precise?

Reply:
the composables that are doing fetches to the API could maybe grouped into their own folders, like service methods